docs/source/savant_101/40_cm.rst

```diff
@@ -31,6 +31,10 @@ It is important to note that when ``element.model.output.converter`` is specifie
 
 If the converter is not specified, the model will be configured as a regular ``nvinfer`` classifier. In this case, the optional configuration parameters for the attributes listed above (except ``threshold``) are not used, and custom parsing of classifier results must be done with Deepstream functionality by defining the ``element.model.custom_lib_path`` and ``element.model.parse_classifier_func_name`` parameters in the configuration. If a custom function is not specified, Deepstream uses the default ``softmax`` layer function to parse the classification results.
 
+.. note::
```

Comment:
I think it is better to add this note to the paragraph above that talks about the converter.